packages/web_pointer_interceptor/lib/src/web.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore: avoid_web_libraries_in_flutter
+import 'dart:html' as html;
+
+import 'package:flutter/widgets.dart';
+
+import 'shim/dart_ui.dart' as ui;
+
+const String _viewType = '__webPointerInterceptorViewType__';
+const String _debug = 'debug__';
+
+String _getViewType({bool debug = false}) {
+  return debug ? _viewType + _debug : _viewType;
+}
+
+void _registerWrapper({bool debug = false}) {
+  final String viewType = _getViewType(debug: debug);
+
+  ui.platformViewRegistry.registerViewFactory(viewType, (int viewId) {
+    final html.Element wrapper = html.DivElement();
```

Comment:
nit: is "wrapper" a logical name? This element is empty, no?

Reply:
This is totally a leftover from the first experiments I did :P